org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/LanguageClient.java

```diff
@@ -141,4 +143,23 @@
 	default void semanticHighlighting(SemanticHighlightingParams params) {
 		throw new UnsupportedOperationException();
 	}
+
+	/**
+	 * This request is sent from the server to the client to ask the client to create a work done progress.
+	 */
+	@JsonNotification("window/workDoneProgress/create")
+	default void createProgress(WorkDoneProgressCreateParams params) {
```

Comment:
This should be a `@JsonRequest`, and the return type should be `CompletableFuture<Void>`.

Reply:
It's specified as a request here:
https://microsoft.github.io/language-server-protocol/specifications/specification-current/#window_workDoneProgress_create